### ISSUE TITLE
[Bug] NoClassDefFoundError: org/postgresql/jdbc/TimestampUtils$ParsedTimestamp

### ISSUE THREAD

**Issue by starudream** [apache/doris#17227]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.

### Version

[1.2.2-rc01](https://github.com/apache/doris/tree/1.2.2-rc01)

### What's Wrong?

```
mysql> SELECT * FROM test_postgres.public.t1;
ERROR 1105 (HY000): errCode = 2, detailMessage = NoClassDefFoundError: org/postgresql/jdbc/TimestampUtils$ParsedTimestamp
```

### What You Expected?

no error

### How to Reproduce?

- pg

```shell
docker run --name pg -e POSTGRES_PASSWORD=123456 -p 5432:5432 -d postgres:13
```

```sql
CREATE TABLE t1
(
    id    INT,
    ts    TIMESTAMP,
    ts_tz TIMESTAMPTZ
);

INSERT INTO t1 (id, ts, ts_tz)
VALUES (1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
```

- doris

```sql
CREATE RESOURCE test_postgres PROPERTIES (
    "type" = "jdbc",
    "user" = "postgres",
    "password" = "123456",
    "jdbc_url" = "jdbc:postgresql://127.0.0.1:5432/postgres",
    "driver_url" = "http://127.0.0.1:8888/postgresql-42.5.4.jar",
    "driver_class" = "org.postgresql.Driver"
);
```

```sql
CREATE CATALOG test_postgres WITH RESOURCE test_postgres;
```

```sql
SELECT * FROM test_postgres.public.t1;
```

### Anything Else?

```log
2023-03-01 02:21:58,124 WARN (thrift-server-pool-3|245) [Coordinator.updateFragmentExecStatus():1733] one instance report fail, query_id=96d496e9c1b14dc0-9d8bf35a1737fd3c instance_id=96d496e9c1b14dc0-9d8bf35a1737fd3d, error message: NoClassDefFoundError: org/postgresql/jdbc/TimestampUtils$ParsedTimestamp
2023-03-01 02:21:58,125 WARN (thrift-server-pool-3|245) [Coordinator.updateStatus():871] one instance report fail throw updateStatus(), need cancel. job id: -1, query id: 96d496e9c1b14dc0-9d8bf35a1737fd3c, instance id: 96d496e9c1b14dc0-9d8bf35a1737fd3d, error message: NoClassDefFoundError: org/postgresql/jdbc/TimestampUtils$ParsedTimestamp
2023-03-01 02:21:58,125 INFO (thrift-server-pool-3|245) [Coordinator.cancelInternal():970] unfinished instance: 96d496e9c1b14dc0-9d8bf35a1737fd3d
2023-03-01 02:21:58,133 WARN (mysql-nio-pool-10|2230) [Coordinator.getNext():891] get next fail, need cancel. query id: 96d496e9c1b14dc0-9d8bf35a1737fd3c
2023-03-01 02:21:58,134 WARN (mysql-nio-pool-10|2230) [Coordinator.getNext():911] query failed: NoClassDefFoundError: org/postgresql/jdbc/TimestampUtils$ParsedTimestamp
2023-03-01 02:21:58,134 WARN (mysql-nio-pool-10|2230) [StmtExecutor.sendResult():1212] cancel fragment query_id:96d496e9c1b14dc0-9d8bf35a1737fd3c cause errCode = 2, detailMessage = NoClassDefFoundError: org/postgresql/jdbc/TimestampUtils$ParsedTimestamp
2023-03-01 02:21:58,134 WARN (mysql-nio-pool-10|2230) [StmtExecutor.execute():589] execute Exception. stmt[40, 96d496e9c1b14dc0-9d8bf35a1737fd3c], errCode = 2, detailMessage = NoClassDefFoundError: org/postgresql/jdbc/TimestampUtils$ParsedTimestamp
```

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by zixuedanxin** [comment]
我这个 muti jdbc catalog 的时候，只能看到4064行数据，你这边遇到了吗

**Comment by starudream** [comment]
> 我这个muti jdbc目录的时候，只能看到4064行数据，你这边遇到了吗

只能 count 出 4064 :joy:

```sql
mysql> SELECT COUNT(*) FROM xxx;
+----------+
| count(*) |
+----------+
|     4064 |
+----------+
1 row in set (0.20 sec)
```

**Comment by zixuedanxin** [comment]
看来jdbc还不成熟，试了好sqlserver oraccle都有不同的错误，等下一个版本吧

**Comment by starudream** [comment]
@zixuedanxin 
`4064` 的问题我找到了 #16326

```shell
SET BATCH_SIZE = 9999;
```

**Comment by starudream** [comment]
同一句 sql 执行多次，有可能会出现 `NoClassDefFoundError` 也有可能会返回结果

**Comment by zhangstar333** [comment]
> 同一句 sql 执行多次，有可能会出现 `NoClassDefFoundError` 也有可能会返回结果

you Doris version is? and could give more info `NoClassDefFoundError ` in be.info or be.out

**Comment by starudream** [comment]
> > 同一句 sql 执行多次，有可能会出现 `NoClassDefFoundError` 也有可能会返回结果
> 
> you Doris version is? and could give more info `NoClassDefFoundError ` in be.info or be.out

```log
I0301 10:21:52.822839   990 task_worker_pool.cpp:1519] successfully report TASK|host=172.25.0.21|port=9020
I0301 10:21:54.783458   389 daemon.cpp:256] OS physical memory 19.29 GB. Process memory usage 552.73 MB, limit 15.43 GB, soft limit 13.89 GB. Sys available memory 17.39 GB, low water mark 1.60 GB, warning water mark 3.20 GB. Refresh interval memory growth 0 B
I0301 10:21:56.704869   992 tablet_manager.cpp:899] find expired transactions for 0 tablets
I0301 10:21:56.705021   992 tablet_manager.cpp:937] success to build all report tablets info. tablet_count=0
I0301 10:21:56.706379   992 task_worker_pool.cpp:1519] successfully report TABLET|host=172.25.0.21|port=9020
I0301 10:21:57.577124   909 olap_server.cpp:713] cooldown producer get tablet num: 0
I0301 10:21:58.053421  1082 fragment_mgr.cpp:631] query_id: 96d496e9c1b14dc0-9d8bf35a1737fd3c coord_addr TNetworkAddress(hostname=172.25.0.21, port=9020) total fragment num on current host: 1
I0301 10:21:58.053676  1082 fragment_mgr.cpp:681] Register query/load memory tracker, query/load id: 96d496e9c1b14dc0-9d8bf35a1737fd3c limit: 2.00 GB
I0301 10:21:58.053784  1082 plan_fragment_executor.cpp:88] PlanFragmentExecutor::prepare|query_id=96d496e9c1b14dc0-9d8bf35a1737fd3c|instance_id=96d496e9c1b14dc0-9d8bf35a1737fd3d|backend_num=0|pthread_id=140082839266880
I0301 10:21:58.054855   593 fragment_mgr.cpp:492] PlanFragmentExecutor::_exec_actual|query_id=96d496e9c1b14dc0-9d8bf35a1737fd3c|instance_id=96d496e9c1b14dc0-9d8bf35a1737fd3d|pthread_id=140086820935232
I0301 10:21:58.054947   593 plan_fragment_executor.cpp:235] PlanFragmentExecutor::open|query_id=96d496e9c1b14dc0-9d8bf35a1737fd3c|instance_id=96d496e9c1b14dc0-9d8bf35a1737fd3d|mem_limit=2.00 GB
I0301 10:21:58.122128   754 vjdbc_connector.cpp:192] JdbcConnector::query has exec success: SELECT "access_time" FROM "access"."access_record" LIMIT 10
W0301 10:21:58.122730   754 jni-util.cpp:193] java.lang.NoClassDefFoundError: org/postgresql/jdbc/TimestampUtils$ParsedTimestamp
	at org.postgresql.jdbc.TimestampUtils.parseBackendTimestamp(TimestampUtils.java:199)
	at org.postgresql.jdbc.TimestampUtils.toTimestamp(TimestampUtils.java:395)
	at org.postgresql.jdbc.PgResultSet.getTimestamp(PgResultSet.java:637)
	at org.postgresql.jdbc.PgResultSet.internalGetObject(PgResultSet.java:241)
	at org.postgresql.jdbc.PgResultSet.getObject(PgResultSet.java:2916)
	at org.apache.doris.udf.JdbcExecutor.getBlock(JdbcExecutor.java:182)
W0301 10:21:58.122813   754 scanner_scheduler.cpp:253] Scan thread read VScanner failed: Internal error: NoClassDefFoundError: org/postgresql/jdbc/TimestampUtils$ParsedTimestamp
W0301 10:21:58.123072   593 fragment_mgr.cpp:395] report error status: Internal error: NoClassDefFoundError: org/postgresql/jdbc/TimestampUtils$ParsedTimestamp to coordinator: TNetworkAddress(hostname=172.25.0.21, port=9020), query id: 96d496e9c1b14dc0-9d8bf35a1737fd3c, instance id: 96d496e9c1b14dc0-9d8bf35a1737fd3d
W0301 10:21:58.125828   593 fragment_mgr.cpp:251] Got error while opening fragment 96d496e9c1b14dc0-9d8bf35a1737fd3d, query id: 96d496e9c1b14dc0-9d8bf35a1737fd3c: Internal error: NoClassDefFoundError: org/postgresql/jdbc/TimestampUtils$ParsedTimestamp
I0301 10:21:58.125929   593 plan_fragment_executor.cpp:632] PlanFragmentExecutor::cancel|query_id=96d496e9c1b14dc0-9d8bf35a1737fd3c|instance_id=96d496e9c1b14dc0-9d8bf35a1737fd3d|reason=3|error message=PlanFragmentExecutor open failed
I0301 10:21:58.130532   593 plan_fragment_executor.cpp:704] Close() fragment_instance_id=96d496e9c1b14dc0-9d8bf35a1737fd3d
I0301 10:21:58.130594   593 query_fragments_ctx.h:56] Deregister query/load memory tracker, queryId=96d496e9c1b14dc0-9d8bf35a1737fd3c, Limit=2.00 GB, CurrUsed=73.12 KB, PeakUsed=51.09 KB
I0301 10:21:59.799278   389 daemon.cpp:256] OS physical memory 19.29 GB. Process memory usage 553.21 MB, limit 15.43 GB, soft limit 13.89 GB. Sys available memory 17.39 GB, low water mark 1.60 GB, warning water mark 3.20 GB. Refresh interval memory growth 0 B
I0301 10:22:04.814954   389 daemon.cpp:256] OS physical memory 19.29 GB. Process memory usage 553.21 MB, limit 15.43 GB, soft limit 13.89 GB. Sys available memory 17.39 GB, low water mark 1.60 GB, warning water mark 3.20 GB. Refresh interval memory growth 0 B
I0301 10:22:04.824674   990 task_worker_pool.cpp:1519] successfully report TASK|host=172.25.0.21|port=9020
```

**Comment by starudream** [comment]
But now I can't reproduce the problem, but a similar problem occurs `java.lang.NoClassDefFoundError: org/postgresql/jdbc/PgResultSet$CursorResultHandler`, I think it's the same problem.


```log
I0310 16:49:13.499743  1160 plan_fragment_executor.cpp:88] PlanFragmentExecutor::prepare|query_id=6736dececf634b08-97642053c9819ba5|instance_id=6736dececf634b08-97642053c9819ba6|backend_num=0|pthread_id=139619959707200
I0310 16:49:13.501332   626 fragment_mgr.cpp:492] PlanFragmentExecutor::_exec_actual|query_id=6736dececf634b08-97642053c9819ba5|instance_id=6736dececf634b08-97642053c9819ba6|pthread_id=139624070153792
I0310 16:49:13.501448   626 plan_fragment_executor.cpp:235] PlanFragmentExecutor::open|query_id=6736dececf634b08-97642053c9819ba5|instance_id=6736dececf634b08-97642053c9819ba6|mem_limit=2.00 GB
I0310 16:49:14.134461   681 vjdbc_connector.cpp:192] JdbcConnector::query has exec success: SELECT "id", "tenant_id", "created_at", "updated_at", "deleted_at", "event_id", "device_code", "design_code", "card_no", "custom_card_no", "user_account", "user_id", "user_name", "department_id", "department_name", "space_code", "access_time", "result", "direction", "comment", "building_code", "building_id", "floor_id", "floor_code" FROM "access"."access_record"
I0310 16:49:14.253479   626 plan_fragment_executor.cpp:704] Close() fragment_instance_id=6736dececf634b08-97642053c9819ba6
I0310 16:49:14.253562   626 query_fragments_ctx.h:56] Deregister query/load memory tracker, queryId=6736dececf634b08-97642053c9819ba5, Limit=2.00 GB, CurrUsed=118.66 KB, PeakUsed=2.11 MB
I0310 16:49:16.032743  1062 tablet_manager.cpp:899] find expired transactions for 0 tablets
I0310 16:49:16.032932  1062 tablet_manager.cpp:937] success to build all report tablets info. tablet_count=24
I0310 16:49:16.035373  1062 task_worker_pool.cpp:1519] successfully report TABLET|host=172.25.0.21|port=9020
I0310 16:49:16.489935   413 daemon.cpp:256] OS physical memory 31.10 GB. Process memory usage 789.90 MB, limit 24.88 GB, soft limit 22.39 GB. Sys available memory 25.46 GB, low water mark 1.60 GB, warning water mark 3.20 GB. Refresh interval memory growth 0 B
I0310 16:49:17.102592  1060 task_worker_pool.cpp:1519] successfully report TASK|host=172.25.0.21|port=9020
I0310 16:49:20.600450  1161 fragment_mgr.cpp:631] query_id: 6b0329d5091d4c31-8f42078ff9907a60 coord_addr TNetworkAddress(hostname=172.25.0.21, port=9020) total fragment num on current host: 1
I0310 16:49:20.600548  1161 fragment_mgr.cpp:681] Register query/load memory tracker, query/load id: 6b0329d5091d4c31-8f42078ff9907a60 limit: 2.00 GB
I0310 16:49:20.600608  1161 plan_fragment_executor.cpp:88] PlanFragmentExecutor::prepare|query_id=6b0329d5091d4c31-8f42078ff9907a60|instance_id=6b0329d5091d4c31-8f42078ff9907a61|backend_num=0|pthread_id=139619951314496
I0310 16:49:20.601651   636 fragment_mgr.cpp:492] PlanFragmentExecutor::_exec_actual|query_id=6b0329d5091d4c31-8f42078ff9907a60|instance_id=6b0329d5091d4c31-8f42078ff9907a61|pthread_id=139623986226752
I0310 16:49:20.601711   636 plan_fragment_executor.cpp:235] PlanFragmentExecutor::open|query_id=6b0329d5091d4c31-8f42078ff9907a60|instance_id=6b0329d5091d4c31-8f42078ff9907a61|mem_limit=2.00 GB
W0310 16:49:20.606506   681 jni-util.cpp:193] java.lang.NoClassDefFoundError: org/postgresql/jdbc/PgResultSet$CursorResultHandler
	at org.postgresql.jdbc.PgResultSet.next(PgResultSet.java:2143)
	at org.apache.doris.udf.JdbcExecutor.hasNext(JdbcExecutor.java:203)
W0310 16:49:20.606792   636 fragment_mgr.cpp:395] report error status: Internal error: NoClassDefFoundError: org/postgresql/jdbc/PgResultSet$CursorResultHandler to coordinator: TNetworkAddress(hostname=172.25.0.21, port=9020), query id: 6b0329d5091d4c31-8f42078ff9907a60, instance id: 6b0329d5091d4c31-8f42078ff9907a61
W0310 16:49:20.609669   636 fragment_mgr.cpp:251] Got error while opening fragment 6b0329d5091d4c31-8f42078ff9907a61, query id: 6b0329d5091d4c31-8f42078ff9907a60: Internal error: NoClassDefFoundError: org/postgresql/jdbc/PgResultSet$CursorResultHandler
I0310 16:49:20.609745   636 plan_fragment_executor.cpp:632] PlanFragmentExecutor::cancel|query_id=6b0329d5091d4c31-8f42078ff9907a60|instance_id=6b0329d5091d4c31-8f42078ff9907a61|reason=3|error message=PlanFragmen
```

**Comment by starudream** [comment]
- doris version: `1.2.2-rc01`
- postgresql jdbc version: `42.5.4`

```
mysql> show frontends\G;
*************************** 1. row ***************************
             Name: 172.25.0.21_9010_1677576826839
               IP: 172.25.0.21
      EditLogPort: 9010
         HttpPort: 8030
        QueryPort: 9030
          RpcPort: 9020
             Role: FOLLOWER
         IsMaster: true
        ClusterId: 929083300
             Join: true
            Alive: true
ReplayedJournalId: 246362
    LastHeartbeat: 2023-03-10 08:51:30
         IsHelper: true
           ErrMsg:
          Version: doris-1.2.2-rc01-Unknown
 CurrentConnected: Yes
1 row in set (0.06 sec)

ERROR:
No query specified

mysql> show backends\G;
*************************** 1. row ***************************
              BackendId: 10003
                Cluster: default_cluster
                     IP: 172.25.0.11
          HeartbeatPort: 9050
                 BePort: 9060
               HttpPort: 8040
               BrpcPort: 8060
          LastStartTime: 2023-03-10 08:38:40
          LastHeartbeat: 2023-03-10 08:51:35
                  Alive: true
   SystemDecommissioned: false
  ClusterDecommissioned: false
              TabletNum: 24
       DataUsedCapacity: 588.061 MB
          AvailCapacity: 441.556 GB
          TotalCapacity: 491.152 GB
                UsedPct: 10.10 %
         MaxDiskUsedPct: 10.10 %
     RemoteUsedCapacity: 0.000
                    Tag: {"location" : "default"}
                 ErrMsg:
                Version: doris-1.2.2-rc01-Unknown
                 Status: {"lastSuccessReportTabletsTime":"2023-03-10 08:51:22","lastStreamLoadTime":-1,"isQueryDisabled":false,"isLoadDisabled":false}
HeartbeatFailureCounter: 0
               NodeRole: mix
1 row in set (0.01 sec)

ERROR:
No query specified
```

**Comment by zhangstar333** [comment]
the NoClassDefFoundError error seems strange，it's sometimes ok, sometimes will be wrong. 
We suspect Doris version 1.2.2-rc01 have problems in load JDBC Driver  class, so we change the way load driver,
If you can, you could be test this in the new version

**Comment by cqyzh** [comment]
> > > 同一句 sql 执行多次，有可能会出现 `NoClassDefFoundError` 也有可能会返回结果
> > 
> > 
> > you Doris version is? and could give more info `NoClassDefFoundError ` in be.info or be.out
> 
> ```
> I0301 10:21:52.822839   990 task_worker_pool.cpp:1519] successfully report TASK|host=172.25.0.21|port=9020
> I0301 10:21:54.783458   389 daemon.cpp:256] OS physical memory 19.29 GB. Process memory usage 552.73 MB, limit 15.43 GB, soft limit 13.89 GB. Sys available memory 17.39 GB, low water mark 1.60 GB, warning water mark 3.20 GB. Refresh interval memory growth 0 B
> I0301 10:21:56.704869   992 tablet_manager.cpp:899] find expired transactions for 0 tablets
> I0301 10:21:56.705021   992 tablet_manager.cpp:937] success to build all report tablets info. tablet_count=0
> I0301 10:21:56.706379   992 task_worker_pool.cpp:1519] successfully report TABLET|host=172.25.0.21|port=9020
> I0301 10:21:57.577124   909 olap_server.cpp:713] cooldown producer get tablet num: 0
> I0301 10:21:58.053421  1082 fragment_mgr.cpp:631] query_id: 96d496e9c1b14dc0-9d8bf35a1737fd3c coord_addr TNetworkAddress(hostname=172.25.0.21, port=9020) total fragment num on current host: 1
> I0301 10:21:58.053676  1082 fragment_mgr.cpp:681] Register query/load memory tracker, query/load id: 96d496e9c1b14dc0-9d8bf35a1737fd3c limit: 2.00 GB
> I0301 10:21:58.053784  1082 plan_fragment_executor.cpp:88] PlanFragmentExecutor::prepare|query_id=96d496e9c1b14dc0-9d8bf35a1737fd3c|instance_id=96d496e9c1b14dc0-9d8bf35a1737fd3d|backend_num=0|pthread_id=140082839266880
> I0301 10:21:58.054855   593 fragment_mgr.cpp:492] PlanFragmentExecutor::_exec_actual|query_id=96d496e9c1b14dc0-9d8bf35a1737fd3c|instance_id=96d496e9c1b14dc0-9d8bf35a1737fd3d|pthread_id=140086820935232
> I0301 10:21:58.054947   593 plan_fragment_executor.cpp:235] PlanFragmentExecutor::open|query_id=96d496e9c1b14dc0-9d8bf35a1737fd3c|instance_id=96d496e9c1b14dc0-9d8bf35a1737fd3d|mem_limit=2.00 GB
> I0301 10:21:58.122128   754 vjdbc_connector.cpp:192] JdbcConnector::query has exec success: SELECT "access_time" FROM "access"."access_record" LIMIT 10
> W0301 10:21:58.122730   754 jni-util.cpp:193] java.lang.NoClassDefFoundError: org/postgresql/jdbc/TimestampUtils$ParsedTimestamp
> 	at org.postgresql.jdbc.TimestampUtils.parseBackendTimestamp(TimestampUtils.java:199)
> 	at org.postgresql.jdbc.TimestampUtils.toTimestamp(TimestampUtils.java:395)
> 	at org.postgresql.jdbc.PgResultSet.getTimestamp(PgResultSet.java:637)
> 	at org.postgresql.jdbc.PgResultSet.internalGetObject(PgResultSet.java:241)
> 	at org.postgresql.jdbc.PgResultSet.getObject(PgResultSet.java:2916)
> 	at org.apache.doris.udf.JdbcExecutor.getBlock(JdbcExecutor.java:182)
> W0301 10:21:58.122813   754 scanner_scheduler.cpp:253] Scan thread read VScanner failed: Internal error: NoClassDefFoundError: org/postgresql/jdbc/TimestampUtils$ParsedTimestamp
> W0301 10:21:58.123072   593 fragment_mgr.cpp:395] report error status: Internal error: NoClassDefFoundError: org/postgresql/jdbc/TimestampUtils$ParsedTimestamp to coordinator: TNetworkAddress(hostname=172.25.0.21, port=9020), query id: 96d496e9c1b14dc0-9d8bf35a1737fd3c, instance id: 96d496e9c1b14dc0-9d8bf35a1737fd3d
> W0301 10:21:58.125828   593 fragment_mgr.cpp:251] Got error while opening fragment 96d496e9c1b14dc0-9d8bf35a1737fd3d, query id: 96d496e9c1b14dc0-9d8bf35a1737fd3c: Internal error: NoClassDefFoundError: org/postgresql/jdbc/TimestampUtils$ParsedTimestamp
> I0301 10:21:58.125929   593 plan_fragment_executor.cpp:632] PlanFragmentExecutor::cancel|query_id=96d496e9c1b14dc0-9d8bf35a1737fd3c|instance_id=96d496e9c1b14dc0-9d8bf35a1737fd3d|reason=3|error message=PlanFragmentExecutor open failed
> I0301 10:21:58.130532   593 plan_fragment_executor.cpp:704] Close() fragment_instance_id=96d496e9c1b14dc0-9d8bf35a1737fd3d
> I0301 10:21:58.130594   593 query_fragments_ctx.h:56] Deregister query/load memory tracker, queryId=96d496e9c1b14dc0-9d8bf35a1737fd3c, Limit=2.00 GB, CurrUsed=73.12 KB, PeakUsed=51.09 KB
> I0301 10:21:59.799278   389 daemon.cpp:256] OS physical memory 19.29 GB. Process memory usage 553.21 MB, limit 15.43 GB, soft limit 13.89 GB. Sys available memory 17.39 GB, low water mark 1.60 GB, warning water mark 3.20 GB. Refresh interval memory growth 0 B
> I0301 10:22:04.814954   389 daemon.cpp:256] OS physical memory 19.29 GB. Process memory usage 553.21 MB, limit 15.43 GB, soft limit 13.89 GB. Sys available memory 17.39 GB, low water mark 1.60 GB, warning water mark 3.20 GB. Refresh interval memory growth 0 B
> I0301 10:22:04.824674   990 task_worker_pool.cpp:1519] successfully report TASK|host=172.25.0.21|port=9020
> ```

感觉就是这个版本有问题，应该怎么解决呢
